force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1950,6 +1950,14 @@
         <shortDescription>Message when EDA can&apos;t access valid product information</shortDescription>
         <value>We couldn&apos;t access valid product information for {0}. Contact Salesforce Customer Support for help.</value>
     </labels>
+    <labels>
+        <fullName>stgErrorReleaseGateActivateErrorResponse</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Message when there was an error during the release gate activation</shortDescription>
+        <value>We couldn&apos;t activate {0}. See Additional details for more information, or contact Salesforce Support for help.&#13;&#10;&#13;&#10;Additional details: {1}</value>
```

Comment:
I was just wondering if the "&#13;&#10;&#13;&#10;" is needed. Is that a return? I didn't see any wrap in the message.

Reply:
The standard toast component doesn't support cr/lf so those will be replaced with space
If we decide to use a messagebox in future cr/lf will be visible

Reply:
oh ok, sounds good. Thanks for explaining that.